manifests/repo.pp

```diff
@@ -32,15 +32,14 @@
 
   case $::osfamily {
     'Debian': {
-      if !defined(Class['apt']) {
-        class { 'apt': }
-      }
+      include ::apt
+      Class['apt::update'] -> Package<| tag == 'elasticsearch-package' |>
```

Comment:
Since repo management is automatically done before package install via init.pp does it make sense to do a notify on `Class['apt::update']` after the `apt::source` part?